CHANGELOG.md

```diff
@@ -29,7 +29,7 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## [5.0.1](https://github.com/asfadmin/Discovery-asf_search/compare/v5.0.0...v5.0.1)
 ### Changed
 - `ASFProduct` is now aware of the session used during search (if available) and will use that by default to download. A session can still be explicitly provided as before.
-- `ASFProduct.stack()` now accepts a `session` argument. The object's own session is used by default.
+- If no opts is provided to `ASFProduct.stack()`, the default will now use the session used to create the product.
```

Comment:
`ASFProduct.stack()` now uses the session provided via the `opts` argument. If none is provided, it will use the session referenced by `ASFProduct.session`